Bottom: 8b58e187aff8156ed1d16d6af530bd0bffab5561
Top:    95fe9b20ac5ad19f5ce17886136d1c7fe1c6d376
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:31:50 +0000

Fix netlist selection to save undo information for wiped "FOUND" flags

When we wipe the found flags on objects, we should also queue them
for redraw - fix that too.


---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index addb0d0..ce3e2ff 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -516,7 +516,7 @@ netlist_select_cb (GtkWidget * widget, gpointer data)
     node_selected_net = NULL;
 
   InitConnectionLookup ();
-  ResetConnections (false);
+  ResetConnections (true);
 
   for (i = selected_net->EntryN, entry = selected_net->Entry; i; i--, entry++)
     if (SeekPad (entry, &conn, false))
diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index a2d3507..4ddc240 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -136,7 +136,7 @@ nbcb_select_common (LibraryMenuTypePtr net, int pos, int select_flag)
   int i;
 
   InitConnectionLookup ();
-  ResetConnections (false);
+  ResetConnections (true);
 
   for (i = net->EntryN, entry = net->Entry; i; i--, entry++)
     if (SeekPad (entry, &conn, false))
